packages/components/package.json

```diff
@@ -11,6 +12,7 @@
     "@types/node": "^17.0.24",
     "@types/react": "^18.0.3",
     "@types/react-dom": "^18.0.0",
+    "@types/styled-components": "^5.1.25",
```

Comment:
`"@types/styled-components": "^5.1.24",` appears in `devDependencies` of this same file.

Reply:
Good notice!